run-build-functions.sh

```diff
@@ -50,6 +50,9 @@ mkdir -p $NETLIFY_CACHE_DIR/.bundle
 mkdir -p $NETLIFY_CACHE_DIR/bower_components
 mkdir -p $NETLIFY_CACHE_DIR/.venv
 mkdir -p $NETLIFY_CACHE_DIR/.build
+
+# If you remove this line, please ensure that $NETLIFY_CACHE_DIR/.netlify is
+# still created, as it is used by the Rust functions cache.
 mkdir -p $NETLIFY_CACHE_DIR/.netlify/plugins
```

Comment:
Could we do a separate `mkdir -p $NETLIFY_CACHE_DIR/.netlify` instead? To be more explicit.

Reply:
Much better! Done in https://github.com/netlify/build-image/pull/602/commits/4ccc5c322e7c09e49717d53a17ac63019c96f34e.